src/lib.rs

```diff
@@ -276,42 +291,41 @@ fn create_pipeline(
     device: Arc<Device>,
     subpass: Subpass,
 ) -> Result<Arc<EguiPipeline>, GraphicsPipelineCreationError> {
-    let vs = shaders::vs::Shader::load(device.clone()).unwrap();
-    let fs = shaders::fs::Shader::load(device.clone()).unwrap();
+    let vs = shaders::vs::load(device.clone()).unwrap();
+    let fs = shaders::fs::load(device.clone()).unwrap();
 
-    let mut blend = AttachmentBlend::alpha_blending();
+    let mut blend = AttachmentBlend::alpha();
     blend.color_source = BlendFactor::One;
 
-    let pipeline = Arc::new(
-        GraphicsPipeline::start()
-            .vertex_input_single_buffer::<Vertex>()
-            .vertex_shader(vs.main_entry_point(), ())
-            .triangle_list()
-            .viewports_scissors_dynamic(1)
-            .fragment_shader(fs.main_entry_point(), ())
-            .cull_mode_disabled()
-            .blend_collective(blend)
-            .render_pass(subpass)
-            .build(device.clone())?,
-    );
+    let pipeline = GraphicsPipeline::start()
+        .vertex_input_state(BuffersDefinition::new().vertex::<Vertex>())
+        .vertex_shader(vs.entry_point("main").unwrap(), ())
+        .input_assembly_state(InputAssemblyState::new())
+        .viewport_state(ViewportState::viewport_dynamic_scissor_irrelevant())
+        .fragment_shader(fs.entry_point("main").unwrap(), ())
+        .rasterization_state(
+            RasterizationState::new()
+                .cull_mode(CullMode::None)
+        )
+        .color_blend_state(ColorBlendState::new(subpass.num_color_attachments()).blend_alpha())
```

Comment:
Found the blending issue:

```suggestion
        .color_blend_state(ColorBlendState::new(subpass.num_color_attachments()).blend(blend))
```

Reply:
Good find, fixed this as well.